### ISSUE TITLE
Histocat failing to identify individual cells

### ISSUE THREAD

**Issue by LeoMv999** [BodenmillerGroup/histoCAT#99]
Hello

I'm interested in profiling immune cells in the gut microenvironment using immunohistochemistry. I am using histocat (version 1.74 on windows 10) for my analysis of images taken from gut tissue biopsies.

I generated a segmentation mask (8bit) using ilastik and uploaded it together with individual images of 4 fluorescent channels representing different markers. The DAPI channel is included in the image set.

When I select the mask checkbox, the segmentation mask is able to accurately segment and highlight individual cells. However, I noticed that when I select the "Plot sample area XY" checkbox, Histocat is only able to pick up the centroid of a small fraction of cells, whereas a large fraction of the cells are left undetected. In addition, some of the blank background regions of the image are being identified as cell centroids by histocat.

I have tried to use 16bit and 32bit image masks, however, histocat is failing to upload the associated image files when I try this. I also checked to ensure all images including the mask are the same size and indeed they are. I am not sure what the issue is and any help would be very much appreciated.

Thanks!

Leonard

**Comment by DenisSch** [comment]
Dear @LeoMv999 

I would suggest the following next steps:
1) Does histoCAT works with the exemplar image: https://github.com/BodenmillerGroup/histoCAT/releases/download/histoCAT_1.76/Example_Images.zip
2) Is your mask integer or float? Float may create issues.
3) Can you share one image + mask for trouble shooting?

Best

Denis

**Comment by LeoMv999** [comment]
Dear Denis,

1. HistoCAT is working with the exemplar image without any issues.

2. The mask is float and not integer

3. I have shared an example image 
[Example_image_leonard.zip](https://github.com/BodenmillerGroup/histoCAT/files/7119607/Example_image_leonard.zip)

**Comment by DenisSch** [comment]
Dear @LeoMv999

Your mask does not persevere the original cell ids. It looks like the mask was processed and blurring was applied.
Two steps:
1. Use tifs instead of bmp
2. Use the original output from CellProfiler (or any other software) without conversion

**Comment by LeoMv999** [comment]
Dear Denis,

I ran the pipeline using your suggestions and it worked perfectly. In addition to the steps you suggested, I used a 16bit image mask instead of the 8bit mask. Although I did notice when I tried to load my images in Histocat it took 12+ hours for the images to load. However, when I selected a smaller ROI within the original images which I ran through the pipeline and loaded in Histocat, it took only a few minutes for the images to load. I am assuming this is because my computer may not have the necessary processing power to run the larger image sizes?
